setup.py

```diff
@@ -157,6 +157,8 @@ def run(self):
       author='A. Jesse Jiryu Davis',
       author_email='jesse@mongodb.com',
       url='https://github.com/mongodb/motor/',
+      python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, != 3.3.*, '
+                      '!=3.5.0, !=3.5.1',
```

Comment:
Can you add `'!=3.4.*'` to the list and wrap the multi-line string in parentheses `('...')`?

Reply:
Reformatted.
3.5 is supported according to requirements.rst...

Reply:
Oh you're totally right!